<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PathFinderConfigWidget</class>
 <widget class="QWidget" name="PathFinderConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>832</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;Select path(s) between two nodes&lt;/b&gt;&lt;/p&gt;&lt;/br&gt;
&lt;p&gt;This interactor computes and selects path(s) between two nodes following several parameters (defined below).&lt;/p&gt;&lt;/br&gt;
&lt;p&gt;&lt;b&gt;Step 1: &lt;/b&gt;Select (click on) the source node.&lt;br/&gt;
&lt;b&gt;Step 2: &lt;/b&gt;Select the target node.&lt;/p&gt;&lt;/br&gt;
&lt;p&gt;If a path exists, it will be selected. Then, if you select (click on) another node, it will become the new target node and a new path will be computed.&lt;/p&gt;&lt;/br&gt;
&lt;p&gt;To reset your current source selection, simply double-click on the new source node. A double-click outside any node will deselect all.&lt;/p&gt;&lt;/br&gt;
&lt;p align=&quot;center&quot; &gt;&lt;b&gt;Settings&lt;/b&gt;&lt;/p&gt;&lt;/br&gt;
&lt;p&gt;To get an explanation of each setting, please refer to the available tooltips on each setting line.&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;b&gt;Weight metric&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="weightCombo">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can select a graph metric which edge values will be considered as weights over the edges.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;b&gt;Edges direction&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="edgeOrientationCombo">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can choose to consider edges as directed, undirected or reversed.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;b&gt;Path type&lt;/b&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="pathsTypeCombo">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Specify the kind of path to compute.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="toleranceLabel">
     <property name="text">
      <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;Max path length&lt;/b&gt;&lt;/p&gt;
&lt;p&gt;When selecting all paths, set the max path length to a % of the shortest path length greater than 100 to be sure to select more than all the shorted paths.&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="toleranceCheck">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Max path length</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="toleranceSpin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimum">
        <number>100</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tolerancePercentLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>% of the shortest path length</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="highlightersLabel">
     <property name="text">
      <string>&lt;p align=&quot;center&quot;&gt;&lt;b&gt;Path highlighters&lt;/b&gt;&lt;/p&gt;
&lt;p&gt;You can choose between one or many methods to highlight paths once they are selected.&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="bottomArea"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
